CHANGELOG.md

```diff
@@ -1,19 +1,44 @@
 # Changelog
 
-## TBD
+## 5.17.0 (2021-12-08)
 
-## Bug fixes
+### Enhancements
 
-* `redactedKeys` now correctly apply to metadata on Event breadcrumbs
-  [#1526](https://github.com/bugsnag/bugsnag-android/pull/1526)
+* Bump compileSdkVersion to apiLevel 31
+  [#1536](https://github.com/bugsnag/bugsnag-android/pull/1536)
 
-## 5.15.0 (2021-11-04)
+### Bug fixes
+
+* Flush in-memory sessions first
+  [#1538](https://github.com/bugsnag/bugsnag-android/pull/1538)
+
+* Avoid unnecessary network connectivity change breadcrumb
+  [#1540](https://github.com/bugsnag/bugsnag-android/pull/1540)
+  [#1546](https://github.com/bugsnag/bugsnag-android/pull/1546)
+
+* Clear native stacktrace memory in `bugsnag_notify_env` before attempting to unwind the stack
+  [#1543](https://github.com/bugsnag/bugsnag-android/pull/1543)
+
+## 5.16.0 (2021-11-29)
 
 ### Bug fixes
 
 * Increase resilience of NDK stackframe method capture
   [#1484](https://github.com/bugsnag/bugsnag-android/pull/1484)
```

Comment:
I think this changelog section should be in 5.15.0 - looks like the sections merged incorrectly.

Reply:
These seem to match up with the release notes on GH for 5.16.0, unless I've got the wrong end of the stick? https://github.com/bugsnag/bugsnag-android/releases/tag/v5.16.0

Reply:
weird - it must be that a previous merge from next into the feature flag integration branch went awry, as [it has it in 5.15.0](https://github.com/bugsnag/bugsnag-android/blob/integration/feature-flags/CHANGELOG.md#5150-2021-11-04)